### PR TITLE
feat(cx-templates, cx-flowcontainer): Add generic template 

### LINKED ISSUE #1463: Generic Template & Flow Container
## Description
As a site builder I want a generic template and flow container to start using immediately so I can create a basic page with basic elements.

## Design:
- SEE JIRA ticket

## Acceptance Criteria
AC1: Canvasx-flowcontainer contains utlitity tokens that do the following:
- Token which constrains a flow container to full width.
- Token which constrains a flow container so that items are 1/3 width on tablet.
- Token which defines a "ContentRegion" (a default nested flow container)  another flow container.
- Token which defines Canvasx gutter between components at different responsive breakpoints: https://xd.adobe.com/view/fd6e4dde-2ecf-480a-aaaf-f5043cb04bf0-a83d/screen/4f1d267f-2649-476d-a1ed-34b0bd125c44/specs/.  
  - Mobile: 0.5rem, Tablet: 1.125rem, Desktop: 1.5rem 

AC2: Canvasx-template contains a generic template 
- Has slots for: Breadcrumb, Top Content, Content, Bottom Content and necessary wrappers.
- Where the Top Content is full width of viewport.
  
AC3: _cxstarter__ package contains a flowcontainer to apply the following base components 
- RTE
- CanvasX Image Variations
- Nested Flow container (with RTE & CanvasX Image variations)

AC3a: _cxstarter__ package contains an example shadowed flowcontainer

AC4: Homepage of the canvasx site is converted to generic template.

AC4. Documentation on canvasx-flowcontainer and generic template.

## Technical Notes:
* May need to move the mx-site-percent set at Layout's container wrapper to template slots.

### PR DESCRIPTION
**Pull request by wodenx**
## Changes
Add generic template and default flow container to cx design system

## Related Issues
Closes #1463

### CODE REVIEW

File: packages/fclasses/src/tokenSpec.ts [hunk 1]
```diff
@@ -70,11 +70,11 @@ function as(
   const args = args$.filter(Boolean);
 
   // Ensure that all token specs have been passed through `asTokenSpec`
-  args.forEach(a => {
-    if (typeof a !== 'function' && typeof a !== 'string' && !a![$TokenSpec]) {
-      throw new Error('All token specifications passed to "as" must be created by a version of "asTokenSpec"');
-    }
-  });
+  // args.forEach(a => {
```
Comment (hvanyo) [inline packages/fclasses/src/tokenSpec.ts:73]: Need to Fix

File: packages/cx-flowcontainer/src/components/FlowContainer/FlowContainerClean.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+// import { withoutHydration } from '@bodiess/hydration';
+import { FlowContainer as FlowContainerClean } from '@bodiless/layouts-ui';
+
+const FlowContainerPreview = () => (
+  <div className="bl-bg-black">Content Region</div>
+);
+
+export default FlowContainerClean;
+
+// @todo add after static replacement merges.
```
Comment (hvanyo) [inline packages/cx-flowcontainer/src/components/FlowContainer/FlowContainerClean.tsx:26]: @wodenx  can we do now, Static code is in this PR -- yes?

Comment (hvanyo) [inline packages/cx-flowcontainer/src/components/FlowContainer/FlowContainerClean.tsx:26]: @wodenx  can we do now, Static code is in this PR -- yes?

File: packages/cx-image/src/components/FlowContainer/index.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { asFluidToken, asMetaToken } from '@bodiless/cx-elements';
```
Comment (hvanyo) [inline packages/cx-image/src/components/FlowContainer/index.ts:1]: oh we should run script to add headers

**Review by hvanyo — DISMISSED** [review]

**Review by hvanyo — DISMISSED** [review]

**Review by TatsianaMiraniuk — APPROVED** [review]

**Review by TatsianaMiraniuk — APPROVED** [review]

**Review by hvanyo — APPROVED** [review]

**Pull request merged**